src/components/ApiKeyListItem.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+// Components
+import FAIcon from "./FAIcon";
+
+const ApiKeyListItem = props => {
+  return (
+    <div className="api-key-item">
+      <div className="api-key">
```

Comment:
since the sibling `<div>` has a className of `api-key-actions`, I would consider naming this `api-key-content` rather than just `api-key`.